bin/glrd

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+cmd=$(basename "${0}")
+args="$*"
+if [ -z "${GARDENLINUX_BUILD_CRE}" ]; then
+    GARDENLINUX_BUILD_CRE="sudo podman"
```

Comment:
why is sudo required here (by default)?

Reply:
This is "comman practice" in other places, too:
https://github.com/gardenlinux/gardenlinux/blob/c8b506dd79bd1fc6f8f10a7bf3e2d2eef625f3a9/bin/start-vm#L98
https://github.com/gardenlinux/gardenlinux/blob/c8b506dd79bd1fc6f8f10a7bf3e2d2eef625f3a9/.github/workflows/platform-test-images.yml#L83
https://github.com/gardenlinux/gardenlinux/blob/c8b506dd79bd1fc6f8f10a7bf3e2d2eef625f3a9/.github/workflows/build_container.yml#L19

Just adopted it. Should be changed globally if we don't need this feature anymore. I am not sure who needs to run this as root.